smoke-test.bash

```diff
@@ -18,4 +18,22 @@ while true; do
   sleep 5
 done
 
-echo "Smoke test succeeded"
+docker compose down
+
+# Testing the l3node
```

Comment:
Lets make it a separate github action workflow run and also test that the staker is working. Either set up a matrix to execute the script with 2 configurations or duplicate the workflow and modify it for the L3. I think both are fine.